tools/ExtensionsMetadataGenerator/test/ExtensionsMetadataGeneratorTests/ExtensionsMetadataGeneratorTests.csproj

```
@@ -7,7 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="ExistingRuntimeAssemblies.txt" />
+    <None Remove="ExistingRuntimeAssemblies-net6.txt" />
```

```suggestion
    <None Remove="ExistingRuntimeAssemblies-*.txt" />
```

Can remove the 2nd line then